packages/core/src/IconContainer/IconContainer.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import { faAtom } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { Meta, StoryObj } from "@storybook/react";
+import { HvIconContainer } from "@hitachivantara/uikit-react-icons";
+
+const meta: Meta<typeof HvIconContainer> = {
+  title: "Components/Icon Container",
```

Comment:
I think we should group this with the other icons, in the "Foundation" category. It might not be the perfect spot, but having this in the middle of the Components makes it harder to find/confusing IMO

Reply:
That seems weird to me because Foundation is more of a documentation section with guides than a component showcase, and in this pr is already a story more in form of a guide that explains the usage of the component in that section. 
I can just rename it to the component name but then in my opinion is harder for a user to know that when using a external icon they need to use the icon container or the component breaks. What do you think?

Reply:
all right - makes sense 👍